src/main/utils.js

```diff
@@ -42,3 +42,14 @@ export const isFile = filepath => {
 export const isMarkdownFile = filepath => {
   return isFile(filepath) && hasMarkdownExtension(filepath)
 }
+
+// creates a directory if it doesn't already exist.
+export const mkdir = function (dirPath) {
+  try {
+    fs.mkdirSync(dirPath)
+  } catch (e) {
+    if (e.code !== 'EEXIST') {
+      throw e
```

Comment:
use `log` method to write error log to `log file`

Reply:
I think following code is the best case, otherwise Mark Text will crash because of no existing directory.

```
log(e)
throw e
```

Reply:
ok